internal/venice-common/src/main/proto/VeniceControllerGrpcService.proto

```diff
@@ -0,0 +1,85 @@
+syntax = 'proto3';
+package com.linkedin.venice.protocols;
```

Comment:
Should we include `controller` in the package name? Either in the last or second to last position?

Reply:
should be fine; just need to coordinate the change w/ @sushantmane

Reply:
We are going to have client/server (or "data plane") protocols as well. It would seem cleaner to me to have a clear separation between control plane and data plane protocols.

Reply:
Yeah this fine. I've already made this change in my PR: https://github.com/linkedin/venice/pull/1396. Could you please pickup updated proto files?